sql/core/src/test/scala/org/apache/spark/sql/ArrowSuite.scala

```diff
@@ -77,7 +90,16 @@ class ArrowSuite extends SharedSQLContext {
     collectAndValidate(lowerCaseData, "test-data/arrow/lowercase-strings.json")
   }
 
-  test("time and date conversion") { }
+  ignore("time and date conversion") {
```

Comment:
FYI, I am adding support for time and date. I will rebase on this and finish this test.